index.rst

```diff
@@ -367,93 +404,44 @@ clicking on the "commits" section of the search results, as shown in :ref:`the s
-    This includes, but is not limited to, EFD queries, Jupyter notebooks, other data analysis routines (regardless of the programming language) and so on.
-    The preferred location for storing Jupyter notebooks is the `ts_notebooks <https://github.com/lsst-ts/ts_notebooks>`__ repository.
-
-    ..    Details on how to deals with Camera and DM components will be given in the
-    ..    future.
-
-    Any intermediate data product(s) generated in the process should also be stored in the `git Large File Storage <https://developer.lsst.io/git/git-lfs.html>`__  or, if size permits, with the software repository itself.
-
-#.  Edit/Add/Replace the configuration file(s) or add a new file(s) to host the new configuration in the CSC configuration directory.
-
-        - Ideally the name of the file should reflect the purpose of change, dates can also be used as well.
-          Old configuration files can be kept in the repo if they still represent valid configurations otherwise, they should be removed.
-          Note, though, that they will still remain available on previous versions in the git repo, enabling historical comparison.
-
-#.  Add a (commented out) description in the file detailing where any auxiliary data may be stored, the Jira ticket number used to create the file, and the reason for creating the configuration, such as in `this example <https://tstn-017.lsst.io/v/PREOPS-27/_downloads/ATSpectrograph_example_config.yaml>`__, with the appropriate information.
-
-#.  Modify the configuration labels to either re-use a previous label to map to the new configuration (preferred) or create a new label for the new configuration.
-
-        - For Salobj CSCs, this is done by editing the ``_labels.yaml`` file.
-
-#.  Add, commit and push the changes, with a commit message.
+.. code-block:: python
 
-    .. prompt:: bash
+    from lsst.ts.observatory.control import ATCS
 
-        git commit -am "Add new LUTs for ATAOS (file 20200512-configuration.yaml) based on data taken on 20200512. Check DM-12345 for more information."
-        git push
+    atcs = ATCS()
 
-#.  Test the new configuration on the CSC.
-    If this requires in-dome or on-sky testing, make sure the test is properly documented in a technote and/or Jira ticket.
-    To make the configuration available on a running CSC check :ref:`section-on-the-fly-config`.
+    await atcs.start_task
 
-#.  Create pull request(s) (PRs), with evidence that the  configuration is tested, verified and documented.
+    # ATAOS must be in STANDBY state for this to work. All other CSCs will
+    # use their default configurations
+    await atcs.enable(configurationOverride={'ATAOS': ':c89ef1a'})
 
-    PRs must be created for all repositories that where modified during the process, including, but not limited to, the configuration repository, ancillary software and documentation.
+If we also want to specify an override file then we insert the filename before the colon (``:``) as shown below:
```

Comment:
was it before or after?

Reply:
I thought before..
![image](https://user-images.githubusercontent.com/14132079/116308693-af037000-a75c-11eb-9067-a6c80396eee7.png)